modules/cloud-router-connection/main.tf

```diff
@@ -109,9 +126,8 @@ resource "equinix_fabric_connection" "secondary_cloud_router_connection" {
   a_side {
     access_point {
       type = var.aside_ap_type
-
       router {
-        uuid = var.aside_sec_fcr_uuid
```

Comment:
Why is this being removed? We should be able to specify a different cloud router for the secondary connection. Additionally, this is a breaking change to other examples that are using it.

Reply:
The FCR-2-NE example supports using only one FCR, so we will have to remove the second FCR variable